cmake/templates/conf.py.in

```diff
@@ -517,7 +517,7 @@ master_doc = 'index'
 
 # General information about the project.
 project = u'HPX'
-copyright = u'2008-2020, The STE||AR Group'
+Copyright = u'2008-2020, The STE||AR Group'
```

Comment:
While you're at it, could you change this to `2021`, please

Reply:
Done!